### ISSUE TITLE
-DBOUT_USE_PETSC vs -DPETSc_ROOT

### ISSUE THREAD

**Issue by dschwoerer** [boutproject/BOUT-dev#2510]
One is with capital `C` the other with small `c` - this is confusing.

Can I just rename one? If so, which?

Alternatively we could implement #2380

**Comment by ZedThree** [comment]
Good point, probably all caps is more consistent. Might be nice to do the same for netcdf as well.

#2380 is definitely a nice idea too
